guide/xml/portfile-subports.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V5.0//EN"
+"http://docbook.org/xml/5.0/dtd/docbook.dtd">
+<section xml:id="reference.subports" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Subports</title>
+
+  <para>The subport declaration causes MacPorts to define an additional port,
+  with the <literal>name</literal> given by the declaration. The keywords
```

Comment:
Confusion may arise here due to the use of the term `name`. The `$name` variable does not change in a subport. (And indeed that is the reason for the `dist_subdir` behaviour noted below, because the default value of that is `${name}`.)